### PR TITLE
Add message about restarting node

### PR DESCRIPTION
**Pull request by emord**
@czue I think this is what happened with es2 earlier. Thoughts on adding a message to warn the restarter they might need to go and do things manually?

cc @proteusvacuum

### CODE REVIEW

**Comment by czue** [comment]
I think adding a message about manual intervention is a good idea

File: ansible/roles/elasticsearch/tasks/rolling_restart.yml [hunk 1]
```diff
@@ -18,6 +18,8 @@
   sudo: true
   service: name=elasticsearch state=started
 
+- debug: msg="Sometimes we try to start the node too soon. If hung start node manually"
```
Comment (czue) [inline ansible/roles/elasticsearch/tasks/rolling_restart.yml]: what does this do? (not familiar with it)

Comment (emord) [inline ansible/roles/elasticsearch/tasks/rolling_restart.yml]: the output is 

```
TASK [debug] *******************************************************************
ok: [192.168.33.16] => {
    "msg": "Sometimes we try to start the node too soon. If hung start node manually"
}
```

seemed like the easiest way to add a message, but i didn't look super hard either

Comment (czue) [inline ansible/roles/elasticsearch/tasks/rolling_restart.yml]: neat

**Pull request merged**